kong/tools/wrpc/proto.lua

```diff
@@ -0,0 +1,164 @@
+local pb = require "pb"
```

Comment:
It seems there are no tests for these features

Reply:
Is this a comment for this line or for the whole wRPC feature? I think I'd add tests for wRPC later in another PR, to keep PR size not that large.